docs/en/appium-setup/real-devices-ios.md

```diff
@@ -0,0 +1,280 @@
+the capabilities that you can use.
+
+### Required Software
+
+Appium iOS real device support depends on a central third-party software suite,
+[libimobiledevice](http://www.libimobiledevice.org/), which is easily installable
+with [Homebrew](http://brew.sh/))
+```
+brew install libimobiledevice
+```
+
+## Configuring iOS Real Device Tests: Xcode 7 and iOS 9.3 or below
+
+Automating real devices with iOS 9.3 or below, using Xcode 7.3 or below (i.e.,
+_Instruments_-based testing) requires a relatively small amount of configuration.
+The central requirement is to have a build of your app signed with a development
+provisioning profile. Unfortunately Apple has removed all of its documentation
+on using older versions of Xcode, but a good overview of the process can be
+found [here](https://medium.com/ios-os-x-development/ios-code-signing-provisioning-in-a-nutshell-d5b247760bef#.5hirl92tn)
+and [here](https://engineering.nodesagency.com/articles/iOS/Understanding-code-signing-for-iOS-apps/).
+
+In a little more detail, to get started on a real device, you will need the following:
+
+* An [Apple Developer ID](https://developer.apple.com/programs/ios/) and a valid
+Developer Account with a configured development certificate and provisioning
+profile.
+* A valid iOS Development Certificate and Provisioning Profile are necessary to
+test on a real device. Your app will also need to be signed. You can find
+information about this in the [Apple documentation](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/TestingYouriOSApp/TestingYouriOSApp.html).
+* An iPad or iPhone. Make sure this has been set up for development in Xcode.
+* A signed `.app` or `.ipa` file of your app, or the source code to build one.
+* A Mac with [Xcode](https://developer.apple.com/xcode/) and the Xcode Command
+Line Developer Tools.
+
+Appium handles installing the application to the device, using `ideviceinstaller`
+(part of `libimobiledevice`), but it is sometimes easier to pre-install your app
+using Xcode to ensure there are no problems (see the [iOS deploy](ios-deploy.md)
+document for more information).
+
+### Troubleshooting ideas
+
+0. Make sure UDID is correct by checking it in Xcode Organizer or iTunes. It
+   is a long string (20+ chars).
+0. Make sure that you can run your tests against the Simulator.
+0. Double check that you can invoke your automation from Instruments.
+0. Make sure Instruments is not already running.
+0. Make sure UI Automation is enabled on your device. Settings -> Developer -> Enable UI Automation
+
+
+
+## Configuring iOS Real Device Tests: Xcode 8 and iOS 9.3 and above
+
+In addition to the dependency on `libimobiledevice`, Appium support for real
+devices running iOS 9.3 and above using Xcode 8+ also depends on `ios-deploy`,
+which is easily available through [npm](https://www.npmjs.com/package/ios-deploy)
+```
+npm install -g ios-deploy
```

Comment:
I just tried. I think that the issue was that it is a package that compiles on install. We could put it in optional dependencies, and change the wording here.